ci/travis/ci.sh

```diff
@@ -111,15 +136,29 @@ build() {
 
     # This command should be kept in sync with ray/python/README-building-wheels.md,
     # except the "${MOUNT_BAZEL_CACHE[@]}" part.
-    "${WORKSPACE_DIR}"/ci/suppress_output docker run --rm -w /ray -v "${PWD}":/ray "${MOUNT_BAZEL_CACHE[@]}" -e TRAVIS_COMMIT="${TRAVIS_COMMIT}" -ti rayproject/arrow_linux_x86_64_base:python-3.8.0 /ray/python/build-wheel-manylinux1.sh
+    suppress_output docker run --rm -w /ray -v "${PWD}":/ray "${MOUNT_BAZEL_CACHE[@]}" -e TRAVIS_COMMIT="${TRAVIS_COMMIT}" -ti rayproject/arrow_linux_x86_64_base:python-3.8.0 /ray/python/build-wheel-manylinux1.sh
   fi
 
   if [ "${MAC_WHEELS-}" = 1 ]; then
     # This command should be kept in sync with ray/python/README-building-wheels.md.
-    "${WORKSPACE_DIR}"/ci/suppress_output "${WORKSPACE_DIR}"/python/build-wheel-macos.sh
+    suppress_output "${WORKSPACE_DIR}"/python/build-wheel-macos.sh
+  fi
+}
+
+run() {
+  local result=0
+  ##### BEGIN TASKS #####
```

Comment:
? what's this section about

Reply:
It's where I'm adding code to run tests, build wheel, etc. I have that coming in the next PR.

Reply:
- Then let's add it in the next PR, make this one cleaner. 
- asking again, any chance we can move these to python?

Reply:
- Sure

- I can do that once we've factored out the code from Travis, but rewriting in Python seems orthogonal to being able to call it from GitHub Actions? And if I try to rewrite everything in Python at the same time as I'm moving the scripts around, it's very likely something is going to break. We can revisit that idea once the reshuffling is done?